sorc/ncep_post.fd/FDLVL.f

```diff
@@ -39,13 +39,13 @@
 !> 2002-01-15 | Mike Baldwin | WRF version           
 !> 2011-12-14 | Sarah Lu     | Add GOCART aerosol AERFD
 !> 2021-10-15 | JESSE MENG   | 2D DECOMPOSITION
+!> 2022-09-22 | Li(Kate) Zhang   | Remove Dust=> AERFD
 !>
 !> @author Russ Treadon W/NP2 @date 1992-12-22
       SUBROUTINE FDLVL(ITYPE,TFD,QFD,UFD,VFD,PFD,ICINGFD,AERFD)
```

Comment:
@zhanglikate You may remove "AERFD" in this line, line 68, and lines 101-111.

Reply:
@WenMeng-NOAA  Updated.